services/provider/server/consumer.go

```diff
@@ -224,3 +224,18 @@ func (c *ocsConsumerManager) Get(ctx context.Context, id string) (*ocsv1alpha1.S
 
 	return consumerObj, nil
 }
+
+func (c *ocsConsumerManager) UpdateConsumerReportedStatus(ctx context.Context, id string) error {
```

Comment:
Let's simplify the name: 
- The actual action doesn't care where the status came from so it makes no sense to have the word `Reported` in the name
- Consumer is given as this is a consumer wrapping class. 
- This func is just updating the LastHeartbeat it makes sense to have that in the name

```suggestion
func (c *ocsConsumerManager) UpdateStatusLastHeatbeat(ctx context.Context, id string) error {
```

Reply:
Updated in the latest commit